ttnn/cpp/ttnn/operation.hpp

```diff
@@ -427,8 +429,6 @@ auto default_create_output_tensors(
     return output_tensors;
 }
 
-}  // namespace detail
-
```

Comment:
Reason for moving it out of `detail` namespae?

Reply:
I moved `default_create_output_tensors` out of detail namespace, so I can use it in matmul.